docs/IE_DG/supported_plugins/CL_DNN.md

```diff
@@ -115,6 +115,7 @@ When specifying key values as raw strings (that is, when using Python API), omit
 | `KEY_CLDNN_SOURCES_DUMPS_DIR` | `"<dump_dir>"`                       | `""`               | Final optimized clDNN OpenCL sources dump output directory                                   |
 | `KEY_GPU_THROUGHPUT_STREAMS`  | `KEY_GPU_THROUGHPUT_AUTO`, or positive integer| 1 | Specifies a number of GPU "execution" streams for the throughput mode (upper bound for a number of inference requests that can be executed simultaneously).<br>This option is can be used to decrease GPU stall time by providing more effective load from several streams. Increasing the number of streams usually is more effective for smaller topologies or smaller input sizes. Note that your application should provide enough parallel slack (e.g. running many inference requests) to leverage full GPU bandwidth. Additional streams consume several times more GPU memory, so make sure the system has enough memory available to suit parallel stream execution. Multiple streams might also put additional load on CPU. If CPU load increases, it can be regulated by setting an appropriate `KEY_CLDNN_PLUGIN_THROTTLE` option value (see above). If your target system has relatively weak CPU, keep throttling low. <br>The default value is 1, which implies latency-oriented behavior.<br>`KEY_GPU_THROUGHPUT_AUTO` creates bare minimum of streams to improve the performance; this is the most portable option if you are not sure how many resources your target machine has (and what would be the optimal number of streams). <br> A positive integer value creates the requested number of streams. |
 | `KEY_EXCLUSIVE_ASYNC_REQUESTS` | `YES` / `NO`                | `NO`              | Forces async requests (also from different executable networks) to execute serially.|
+| `KEY_CLDNN_ENABLE_LOOP_UNROLLING` | `YES` / `NO`             | `YES`             | Turning this key on enables to unroll recurrent layers such as TensorIterator or Loop with fixed iteration count. By default, it is turned on. Note that unrolling loops will cause increase of clDNN build time as much as the iteration count, becuase the kernels consiting a body network are created and built for all iterations. On the other hand, disabling unrolling for loops with small iteration count and small body network might perform worse than unrolling, because the overhead for handling subnetworks might be more significant in overall performance. Thus, this key should be set by considering the iteration count and body network's slze.|
```

Comment:
Turning this key on enables to unroll recurrent layers such as TensorIterator or Loop with fixed iteration count. -> Enables recurrent layers such as TensorIterator or Loop to be unrolled with fixed iteration count.
kernels consisting a body network -> kernels in a body network
Thus, this key should be set by considering the iteration count and body network's slze. -> redundant desciption? Or, better to describe the condition you propose?

Reply:
> Thus, this key should be set by considering the iteration count and body network's slze. -> redundant desciption? Or, better to describe the condition you propose?

Actually, the "condition" is not quite clear at this moment because we are still work in progress in optimization. 
I will update or remove the sentence once the optimiziation is finished.

Reply:
Updated !